rpcserver.go

```diff
@@ -1472,6 +1472,16 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 func (r *rpcServer) ListChannels(ctx context.Context,
 	in *lnrpc.ListChannelsRequest) (*lnrpc.ListChannelsResponse, error) {
 
+	if in.ActiveOnly && in.InactiveOnly {
+		return nil, fmt.Errorf("either `active_only` or " +
```

Comment:
Not exactly right, you should be able to not set any of them :)

Reply:
Fixed.